### ISSUE TITLE
At least packages.microsoft.com has 8.0.7 version

### ISSUE THREAD

**Issue by asvishnyakov** [dotnet/runtime#104145]
Looks like you accidentally "released" .NET Runtime 8.0.7 a few weeks (!) ago.
Just see https://packages.microsoft.com/ubuntu/22.04/prod/pool/main/d/dotnet-runtime-8.0/
This version also looks broken, as my apps wasn't unable to find .NET Runtime until I run this:
```bash
sudo apt remove --purge dotnet* aspnet* netstandard*
sudo apt install dotnet-sdk-8.0
```

**Comment by asvishnyakov** [comment]
As a temporary solution I've added
```git
Package: dotnet* aspnet* netstandard*
Pin: version 8.0.7-1
Pin-Priority: -10
```
to `/etc/apt/preferences`

**Comment by huoyaoyuan** [comment]
Also #103622

**Comment by leecow** [comment]
The 8.0.7 installers were accidentally published for 22.04 following the [8.0.302 SDK](https://github.com/dotnet/core/blob/main/release-notes/8.0/8.0.6/8.0.302.md) release. As you know, we typically release updates monthly but did not have a "full" update in June, and anytime you step out of the well-worn path, unexpected things can happen. 8.0.7 will release in July for all platforms and to the usual endpoints.

**Comment by saichadu1702** [comment]
Hey guys, is there any solution for this issue 
"This version also looks broken, as my apps wasn't unable to find .NET Runtime until I run this"
i think Microsoft.WindowsDesktop.App.runtimeconfig.json file is empty because of this , it is not able to detect .Net runtime
